lib/shoppertrak_api_client.py

```diff
@@ -49,9 +49,7 @@ def query(self, endpoint, query_date, query_count=1):
             ) from None
 
         response_root = self._check_response(response.text)
-        if response_root == "E104":
-            return None
-        elif response_root == "E108":
+        if response_root == "E108":
```

Comment:
General question: do we have any documentation on all of the error states that ShopperTrak can return?

Reply:
ShopperTrak sent us some documentation with the following error codes, which notably does not include E104, making me wonder whether there are other possible error codes not included in this list:

- E101: The Site ID supplied is not recognized by the system.
- E102: The format of the “date” value is not valid. The format must be YYYYMMDD.
- E103: The requested date value is older than 30 days.
- E105: The total property value is not valid. This value must either be “Y” or “N”.
- E106: The increment value is not valid. This value must either be “15” or “60”.
- E107: Customer has exceeded the maximum number of requests allowed in a 24 hour period.
- E108: The web service is busy, please try again later. It is recommended to wait at least 30 seconds before making another request.